JenkinsfileProductionRelease

```diff
@@ -2,18 +2,7 @@ node('linux && docker') {
   checkout scm
 
   withDockerContainer(image: 'node:14', args: '-u=root') {
-    stage('Checkout') {
-      sh "git checkout ${params.COMMIT_HASH}"
```

Comment:
I removed the COMMIT_HASH checkout in prod because it does not seem to be needed? The steps seem independent of commit hash?